packages/create-fuels/src/cli.ts

```diff
@@ -30,11 +30,16 @@ const processWorkspaceToml = (fileContents: string) => {
   return toml.stringify(parsed);
 };
 
-function writeEnvFile(envFilePath: string) {
+function writeEnvFile(envFilePath: string, template: Template) {
   let newFileContents = '';
 
-  newFileContents += `\nNEXT_PUBLIC_FUEL_NODE_PORT=4000`;
-  newFileContents += `\nNEXT_PUBLIC_DAPP_ENVIRONMENT=local`;
+  const prefixes: Record<Template, string> = {
+    nextjs: 'NEXT_PUBLIC_',
+    vite: 'VITE_',
+  };
+
+  newFileContents += `\n${prefixes[template]}FUEL_NODE_PORT=4000`;
+  newFileContents += `\n${prefixes[template]}DAPP_ENVIRONMENT=local`;
```

Comment:
Would it be worth replacing strings within the `env` file, such as so then it's template agnostic:

```env
VITE_FUEL_NODE_PORT={{FUEL_NODE_PORT}}
VITE_DAPP_ENVIRONMENT={{DAPP_ENVIRONMENT}}
```

```ts
envFileContents
  .replace(`{{FUEL_NODE_PORT}}`, 4000)
  .replace(`{{DAPP_ENVIRONMENT}}`, 'local')
```


Or we can just have the envs in the `env` file, then we can just copy over `env` -> `.env`?